src/pages/Test.css

```
@@ -1,4 +1,4 @@
-/* Container */
+Container
```

Stylelint (17.3.0)

[error] Expected "Container" to be "container" (selector-type-case)

(selector-type-case)